src/Api.Rest.Dtos.Tests/Benchmarks/InspectionPlanSerializationBenchmark.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Zeiss.PiWeb.Api.Rest.Dtos.Tests.Benchmarks
+{
+	#region usings
+
+	using System;
+	using System.Collections.Generic;
+	using System.Diagnostics.CodeAnalysis;
+	using System.IO;
+	using BenchmarkDotNet.Attributes;
+	using BenchmarkDotNet.Configs;
+	using BenchmarkDotNet.Running;
+	using Newtonsoft.Json;
+	using NUnit.Framework;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Converter;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+
+	/// <summary>
+	/// This class contains various benchmarks for testing the inspection plan serialization/deseialization.
+	/// </summary>
+	/// <remarks>
+	/// Please note: Run these benchmarks in Release mode for reliable and realistic performance numbers.
+	/// </remarks>
+	[TestFixture]
+	[SuppressMessage( "Performance", "CA1822:Member als statisch markieren" )]
+	[MemoryDiagnoser]
+	public class InspectionPlanSerializationBenchmark
```

Comment:
Why is this a separate benchmark class and not included in SerializationBenchmark.cs? If I see it correctly the other class also does the serialization of characteristics.